website/documentation/content/storage_documentation.py

```diff
@@ -86,3 +92,47 @@ def ui_state():
     #         .classes('w-full').bind_value(app.storage.user, 'note')
     # END OF DEMO
     ui.textarea('This note is kept between visits').classes('w-full').bind_value(app.storage.user, 'note')
+
+
+@doc.demo('Short-term memory', '''
+          The goal of `app.storage.client` is to store data only for the duration of the current page visit and
+          only as long as the client is connected.
+          In difference to data stored in `app.storage.tab` - which is persisted between page changes and even 
+          browser restarts as long as the tab is kept open - the data in `app.storage.client` will be discarded 
+          if the user closes the browser, reloads the page or navigates to another page.
+          This is may be beneficial for resource hungry or intentionally very short lived data such as a database 
+          connection which should be closed as soon as the user leaves the page, sensitive data or if you on purpose
+          want to return a page with the default settings every time the user reloads the page while keeping the
+          data alive during in-page navigation or when updating elements on the site in intervals such as a live feed.
+          ''')
+def tab_storage():
+    from nicegui import app, context
+
+    class DbConnection:  # dummy class to simulate a database connection
```

Comment:
The demo is to complicated in my opinion. A simple counter like for `app.storage.tab` would be better to demonstrate the mechanism.

Reply:
I wanted to give the user a better impression about when to use what. Simplified the demo.